lib/librte_hash/rte_cuckoo_hash.c

```diff
@@ -1160,3 +1160,63 @@ rte_hash_iterate(const struct rte_hash *h, const void **key, void **data, uint32
 
 	return position - 1;
 }
+
+inline int
+rte_hash_get_num_buckets(struct rte_hash *h)
```

Comment:
`rte_hash_get_num_buckets()` is also exported in the header file, so it must move there.

Reply:
It seems that ``rte_hash`` tries to hide the hash table data structure in ``rte_hash.h``, the ``struct rte_hash`` is declared in ``rte_cuckoo_hash.h`` file, so if we moved this and below functions to the header file, then we may have to declare the hash table structure in that header file as well, as these two functions need it.

Reply:
In this case, we need to drop the `inline` tag. Thank you for investigating this.